ops/generate-subgraph.js

```diff
@@ -27,6 +27,7 @@ async function generateSubgraph() {
 		return {
 			kind: 'ethereum/contract',
 			name: `${contract}`,
+			network: 'mainnet',
```

Comment:
I guess that 'mainnet' should be replaced with '${network}' as the subgraph suppose to run not just over mainnet .

Reply:
Yeah that makes a lot of sense. @bohendo I looked around briefly and I couldn't figure out the best place to get a `${network}` variable to use in the codegen. Do you have any suggestions? Maybe getting it when the `migration.json` file is created?

Reply:
@davekaj I would suggest adding that to `settings.js`.